### PR TITLE
Add credit and point to hire us

### PR DESCRIPTION
**Pull request by kylefiedler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,3 +158,27 @@ to your own board to help guide and document your sprint.
...
+## Contributing
+
+We love new ideas that push this repository and design sprints forward. Please
+review the [contributing guidelines](CONTRIBUTING.md) if you'd like to help out.
+
+## Credits
+
+[![thoughtbot](http://images.thoughtbot.com/bourbon/thoughtbot-logo.svg)](http://thoughtbot.com)
+
+Design Sprints Repo is maintained and funded by [thoughtbot, inc](http://thoughtbot.com).
```
Comment (gabebw) [inline README.md]: "This repo is maintained..."?

File: README.md [hunk 2]
```diff
@@ -158,3 +158,27 @@ to your own board to help guide and document your sprint.
...
+Tweet your questions or suggestions to
+[@thoughtbot](https://twitter.com/thoughtbot)
+and while you’re at it follow us too.
```
Comment (gabebw) [inline README.md]: normal quotes?

File: README.md [hunk 3]
```diff
@@ -158,3 +158,27 @@ to your own board to help guide and document your sprint.
...
+
+Looking to run your own Design Sprint but want help from someone who has
+experience running them before? [Hire us.](http://thoughtbot.com/hire-us)
+
+## License
+
+Copyright © 2011–2015 [thoughtbot, inc](http://thoughtbot.com).
```
Comment (gabebw) [inline README.md]: Maybe just 2015, since it didn't exist in 2011?

File: README.md [hunk 4]
```diff
@@ -158,3 +158,27 @@ to your own board to help guide and document your sprint.
...
+## Credits
+
+[![thoughtbot](http://images.thoughtbot.com/bourbon/thoughtbot-logo.svg)](http://thoughtbot.com)
+
+This repo is maintained and funded by [thoughtbot, inc](http://thoughtbot.com).
+Tweet your questions or suggestions to
+[@thoughtbot](https://twitter.com/thoughtbot)
+and while you're at it follow us too.
+
+Looking to run your own Design Sprint but want help from someone who has
```
Comment (creuter) [inline README.md]: Design Sprint isn't capitalized elsewhere in the README

**Comment by creuter** [comment]
:+1:

**Pull request closed without merging**